internal/game

```diff
@@ -1 +1 @@
bHVlIjoxLCJpc01pbmUiOmZhbHNlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX0seyJ4cG9zIjo3LCJ5cG9zIjo0LCJ2YWx1ZSI6MCwiaXNNaW5lIjpmYWxzZSwiaXNSZXZlYWxlZCI6ZmFsc2UsImlzRmxhZ2dlZCI6ZmFsc2V9XSxbeyJ4cG9zIjowLCJ5cG9zIjo1LCJ2YWx1ZSI6MCwiaXNNaW5lIjpmYWxzZSwiaXNSZXZlYWxlZCI6ZmFsc2UsImlzRmxhZ2dlZCI6ZmFsc2V9LHsieHBvcyI6MSwieXBvcyI6NSwidmFsdWUiOjIsImlzTWluZSI6ZmFsc2UsImlzUmV2ZWFsZWQiOmZhbHNlLCJpc0ZsYWdnZWQiOmZhbHNlfSx7Inhwb3MiOjIsInlwb3MiOjUsInZhbHVlIjoiOmJvbWI6IiwiaXNNaW5lIjp0cnVlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX0seyJ4cG9zIjozLCJ5cG9zIjo1LCJ2YWx1ZSI6MiwiaXNNaW5lIjpmYWxzZSwiaXNSZXZlYWxlZCI6ZmFsc2UsImlzRmxhZ2dlZCI6ZmFsc2V9LHsieHBvcyI6NCwieXBvcyI6NSwidmFsdWUiOjAsImlzTWluZSI6ZmFsc2UsImlzUmV2ZWFsZWQiOmZhbHNlLCJpc0ZsYWdnZWQiOmZhbHNlfSx7Inhwb3MiOjUsInlwb3MiOjUsInZhbHVlIjowLCJpc01pbmUiOmZhbHNlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX0seyJ4cG9zIjo2LCJ5cG9zIjo1LCJ2YWx1ZSI6MCwiaXNNaW5lIjpmYWxzZSwiaXNSZXZlYWxlZCI6ZmFsc2UsImlzRmxhZ2dlZCI6ZmFsc2V9LHsieHBvcyI6NywieXBvcyI6NSwidmFsdWUiOjAsImlzTWluZSI6ZmFsc2UsImlzUmV2ZWFsZWQiOmZhbHNlLCJpc0ZsYWdnZWQiOmZhbHNlfV0sW3sieHBvcyI6MCwieXBvcyI6NiwidmFsdWUiOjAsImlzTWluZSI6ZmFsc2UsImlzUmV2ZWFsZWQiOmZhbHNlLCJpc0ZsYWdnZWQiOmZhbHNlfSx7Inhwb3MiOjEsInlwb3MiOjYsInZhbHVlIjoyLCJpc01pbmUiOmZhbHNlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX0seyJ4cG9zIjoyLCJ5cG9zIjo2LCJ2YWx1ZSI6Ijpib21iOiIsImlzTWluZSI6dHJ1ZSwiaXNSZXZlYWxlZCI6ZmFsc2UsImlzRmxhZ2dlZCI6ZmFsc2V9LHsieHBvcyI6MywieXBvcyI6NiwidmFsdWUiOjIsImlzTWluZSI6ZmFsc2UsImlzUmV2ZWFsZWQiOmZhbHNlLCJpc0ZsYWdnZWQiOmZhbHNlfSx7Inhwb3MiOjQsInlwb3MiOjYsInZhbHVlIjowLCJpc01pbmUiOmZhbHNlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX0seyJ4cG9zIjo1LCJ5cG9zIjo2LCJ2YWx1ZSI6MCwiaXNNaW5lIjpmYWxzZSwiaXNSZXZlYWxlZCI6ZmFsc2UsImlzRmxhZ2dlZCI6ZmFsc2V9LHsieHBvcyI6NiwieXBvcyI6NiwidmFsdWUiOjAsImlzTWluZSI6ZmFsc2UsImlzUmV2ZWFsZWQiOmZhbHNlLCJpc0ZsYWdnZWQiOmZhbHNlfSx7Inhwb3MiOjcsInlwb3MiOjYsInZhbHVlIjowLCJpc01pbmUiOmZhbHNlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX1dLFt7Inhwb3MiOjAsInlwb3MiOjcsInZhbHVlIjowLCJpc01pbmUiOmZhbHNlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX0seyJ4cG9zIjoxLCJ5cG9zIjo3LCJ2YWx1ZSI6MSwiaXNNaW5lIjpmYWxzZSwiaXNSZXZlYWxlZCI6ZmFsc2UsImlzRmxhZ2dlZCI6ZmFsc2V9LHsieHBvcyI6MiwieXBvcyI6NywidmFsdWUiOjEsImlzTWluZSI6ZmFsc2UsImlzUmV2ZWFsZWQiOmZhbHNlLCJpc0ZsYWdnZWQiOmZhbHNlfSx7Inhwb3MiOjMsInlwb3MiOjcsInZhbHVlIjoxLCJpc01pbmUiOmZhbHNlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX0seyJ4cG9zIjo0LCJ5cG9zIjo3LCJ2YWx1ZSI6MCwiaXNNaW5lIjpmYWxzZSwiaXNSZXZlYWxlZCI6ZmFsc2UsImlzRmxhZ2dlZCI6ZmFsc2V9LHsieHBvcyI6NSwieXBvcyI6NywidmFsdWUiOjAsImlzTWluZSI6ZmFsc2UsImlzUmV2ZWFsZWQiOmZhbHNlLCJpc0ZsYWdnZWQiOmZhbHNlfSx7Inhwb3MiOjYsInlwb3MiOjcsInZhbHVlIjowLCJpc01pbmUiOmZhbHNlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX0seyJ4cG9zIjo3LCJ5cG9zIjo3LCJ2YWx1ZSI6MCwiaXNNaW5lIjpmYWxzZSwiaXNSZXZlYWxlZCI6ZmFsc2UsImlzRmxhZ2dlZCI6ZmFsc2V9XV0sImdhbWVDb3VudCI6MiwibWluZXNGb3VuZCI6MCwiZmFsc2VNaW5lcyI6MCwic3RhdHVzIjoiT05HT0lORyIsIm1vdmVzTWFkZSI6MCwib3B0aW9ucyI6eyJyb3dzIjo4LCJjb2xzIjo4LCJtaW5lcyI6MTB9fQ==
```

Comment:
Oh no! This is just base64 encoded, one could easily decode it and find out the current status of the gameboard